komodoenv/creator.py

```diff
@@ -65,6 +65,19 @@
 if [ -n "${{BASH:-}}" -o -n "${{ZSH_VERSION:-}}" ]; then
     hash -r
 fi
+
+if [ -d {komodo_prefix}/../motd/scripts ]
+then
+    for f in {komodo_prefix}/../motd/scripts/*
```

Comment:
So MOTD ends up in e.g. `/prog/res/komodo/2020.08.01-py36/motd/scripts`?

This could be problematic for matrix builds, where `2020.10-py36` would be a folder with two files in it (just the switch scripts).

Reply:
If `komodo_prefix` is always the concrete build, then that's no problem.

Reply:
you bring up an excellent point

Reply:
but would not komodo_prefix be  /prog/res/komodo/2020.08.01-py36-rhel7, this should not touch the activator script

Reply:
Seems so. If I source `/prog/res/komodo/stable/enable`, `KOMODO_RELEASE` is the concrete build, and I expect `komodo_prefix` to be deduced from that (or similar).